spec/workers/notifications/schedule_date_alerts_notifications_job_spec.rb

```diff
@@ -86,104 +86,95 @@ def deserialize_job(job)
 
   def expect_job(job, *arguments)
     job_data = deserialize_job(job)
-    expect(job_data['job_class']).to eql(job.job_class)
-    expect(job_data['arguments']).to match_array arguments
-    expect(job_data['executions']).to eq 0
+    expect(job_data["job_class"]).to eql(job.job_class)
+    expect(job_data["arguments"]).to match_array arguments
+    expect(job_data["executions"]).to eq 0
   end
 
-  shared_examples_for 'job execution creates date alerts creation job' do
-    let(:timezone) { timezone_paris }
-    let(:scheduled_at) { '1:00' }
-    let(:local_time) { '1:04' }
-    let(:user) { user_paris }
-
-    it 'creates the job for the user' do
+  shared_examples_for "job execution creates date alerts creation job" do
+    it "creates the job for the user" do
       expect do
         run_job(timezone:, scheduled_at:, local_time:) do
           j = GoodJob::Job.where(job_class: "Notifications::CreateDateAlertsNotificationsJob")
                           .order(created_at: :desc)
                           .last
           expect_job(j, user)
         end
-      end.to change(GoodJob::Job, :count).by 1
+      end.to change { GoodJob::Job.where(job_class: "Notifications::CreateDateAlertsNotificationsJob").count }.by 1
```

Comment:
It would be a tiny bit more robust if the job class is taken from the constant `Notifications::CreateDateAlertsNotificationsJob` instead of the string.

Reply:
Good point